cmd/mungedocs/mungedocs.go

```diff
@@ -28,6 +29,8 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+const latestReleaseBranch = "release-1.1"
```

Comment:
Should it get changed in master or on the release branch?  When we branch a new minor version, we auto-commit some versioning changes on the release branch.  See `rev-version-and-commit` in [cut-official-release.sh](https://github.com/kubernetes/kubernetes/blob/master/release/cut-official-release.sh).  Can you do something similar if it's on the release branch?